README.md

```diff
@@ -1,140 +1,103 @@
-
-## Technical documentation
-
-Search API is a Sinatra application that interfaces with Elasticsearch.
-
-There are two ways documents get added to a search index:
-
-1. HTTP requests to Search API's [Documents API](doc/documents.md) (deprecated)
-2. Search API subscribes to RabbitMQ messages from the
-	 [Publishing API](https://github.com/alphagov/publishing-api).
-
-*Note: Once whitehall documents are using the new indexing process, the documents API will be removed and search API will consume only
-from the publishing API.*
-
-Search API search results are weighted by [popularity](doc/popularity.md). We
-rebuild the index nightly to incorporate the latest analytics.
-
-### Nomenclature
-
-- **Link**: Either the base path for a content item, or an external link.
-- **Document**: An elasticsearch document, something we can search for.
-- **Document Type**: An [elasticsearch document
-	type](https://www.elastic.co/guide/en/elasticsearch/guide/current/mapping.html)
-	specifies the fields for a particular type of document. All our document
-	types are defined in
-	[config/schema/elasticsearch_types](config/schema/elasticsearch_types)
-- **Index**: An [elasticsearch search
-	index](https://www.elastic.co/blog/what-is-an-elasticsearch-index). Search API
-	maintains several separate indices (`detailed`, `government` and `govuk`), 
-	but searches return documents from all of them.
-- **Index Group**: An alias in elasticsearch that points to one index at a
-	time. This allows us to rebuild indexes without downtime.
-  
-### Dependencies
+## API documentation
 
-- [elasticsearch](https://github.com/elastic/elasticsearch) - "You Know, for Search...".
-- [redis](https://github.com/redis/redis) - used by indexing workers.
+If you would like to use the Search API, please see the
+[Search API documentation](https://docs.publishing.service.gov.uk/apis/search/search-api.html).
 
-### Creating search indexes from scratch
-(This is not necessary when restoring from a backup or replicating data into the development VM)
+You can also find some examples in the blog post:
+["Use the search API to get useful information about GOV.UK content"](https://gdsdata.blog.gov.uk/2016/05/26/use-the-search-api-to-get-useful-information-about-gov-uk-content/).
 
-To create an empty index:
+## Getting started
 
-    bundle exec rake search:create_index[<index_name>]
+The instructions will help you to get Search API running
+locally on your machine.
 
-To create an empty index for all Search API indices:
+### Dependencies
 
-    SEARCH_INDEX=all bundle exec rake search:create_all_indices
+- [Elasticsearch](https://github.com/elastic/elasticsearch) - "You Know, for Search...".
+- [Redis](https://redis.io/) - used by indexing workers.
+- [AWS Sagemaker](https://aws.amazon.com/sagemaker/) - used for [search relevancy](docs/relevancy.md)
```

Comment:
Sagemaker isn't a dependency for local usage

Reply:
In fact, if you use govuk-docker, you don't need to worry about elasticsearch or redis either